icu4j/maven-migration/toMaven.sh

```diff
@@ -49,24 +67,38 @@ function safeMoveDir() {
     echo "  Error: folder '$TO_FOLDER/$FOLDER_NAME' already exists!"
     exit
   else
-    mkdir -p $TO_FOLDER
-    mv $FROM_FOLDER/$FOLDER_NAME $TO_FOLDER/
+    safeMkdir $TO_FOLDER
+    moveOrRename $FROM_FOLDER/$FOLDER_NAME $TO_FOLDER/
     echo "  Moving '$FOLDER_NAME' folder from '$FROM_FOLDER' to '$TO_FOLDER'"
   fi
 }
 
+function rmFileGlobInDir() {
+  DIR=$1
+  PATTERN=$2
+
+  find "$DIR" -type f      -name "$PATTERN" -exec rm {} \;
```

Comment:
Nitpick: collapse multiple spaces
And comment.

Reply:
Done.